### DISCUSSION TITLE
Cara menggunakan rmdformats: markthedown di RMarkdown

### DISCUSSION THREAD

**Discussion by brigittacarmen** [teamalgoritma/community#709] [Q&A] [answered]
Halo :'D izin bertanya.
Saya punya pertanyaan mengenai cara penggunaan library rmdformats.
Bagaimana cara menuliskan kode yang tepat sehingga bisa berjalan?

rmdformats::material:

Saya telah menuliskan ini dibagian output, tetapi masih execution halted.
Any helps will be highly appreciated, terima kasih!

**Comment by ranydwi** [comment, marked as answer]
Dear @brigittacarmen 

Library rmdformats digunakan untuk mengatur tampilan RMarkdown Bu Brigitta. Apakah betul tujuannya untuk itu? Kalau benar, penggunaan rmdformats silahkan dipanggil terlebih dahulu untuk librarynya 

`library(rmdformats)`

Kemudian pada bagian YAML, silahkan dilakukan pengaturan theme yang ibu inginkan. Saya sudah pernah membuat dokumentasi untuk setiap theme untuk RMarkdown termasuk dari package Rmdformats 
https://rpubs.com/ranydc/rmarkdown_themes

Contoh YAML (detail ada di dalam link rpubs):

```
---
title: "This is a readthedown Themes YAML from `rmdformats` Package"
author: "Type your name here"
date: "`r Sys.Date()`"
output: 
  rmdformats::readthedown
---
```

Semoga membantu Bu.
Regards,
Team Algoritma

**Reply by brigittacarmen** [reply]
Terima kasih atas bantuannya Kak!
